plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendClient.java

```diff
@@ -0,0 +1,745 @@
+    {
+        StringBuilder sb = new StringBuilder()
+                .append(quoted(columnName))
+                .append(" ");
+        if (column.isNullable()) {
+            // set column nullable property explicitly
+            sb.append("Nullable(").append(toWriteMapping(session, column.getType()).getDataType()).append(")");
+        }
+        else {
+            // By default, the clickhouse column is not allowed to be null
+            sb.append(toWriteMapping(session, column.getType()).getDataType());
+        }
+        if (column.getComment() != null) {
+            sb.append(format(" COMMENT %s", databendVarcharLiteral(column.getComment())));
+        }
+        return sb.toString();
+    }
+
+    @Override
+    protected void createSchema(ConnectorSession session, Connection connection, String remoteSchemaName)
+            throws SQLException
+    {
+        execute(session, connection, "CREATE DATABASE " + quoted(remoteSchemaName));
+    }
+
+    @Override
+    protected void dropSchema(ConnectorSession session, Connection connection, String remoteSchemaName, boolean cascade)
+            throws SQLException
+    {
+        if (!cascade) {
+            try (ResultSet tables = getTables(connection, Optional.of(remoteSchemaName), Optional.empty())) {
+                if (tables.next()) {
+                    throw new TrinoException(SCHEMA_NOT_EMPTY, String.format("Cannot drop non-empty schema '%s'", remoteSchemaName));
+                }
+            }
+        }
+        execute(session, connection, "DROP DATABASE " + quoted(remoteSchemaName));
+    }
+
+    @Override
+    protected void renameSchema(ConnectorSession session, Connection connection, String remoteSchemaName, String newRemoteSchemaName)
+            throws SQLException
+    {
+        execute(session, connection, "ALTER DATABASE IF EXISTS " + quoted(remoteSchemaName) + " RENAME TO " + quoted(newRemoteSchemaName));
+    }
+
+    @Override
+    public void addColumn(ConnectorSession session, JdbcTableHandle handle, ColumnMetadata column, ColumnPosition position)
+    {
+        try (Connection connection = connectionFactory.openConnection(session)) {
+            String remoteColumnName = getIdentifierMapping().toRemoteColumnName(getRemoteIdentifiers(connection), column.getName());
+            String sql = format("ALTER TABLE %s ADD COLUMN %s", quoted(handle.asPlainTable().getRemoteTableName()), getColumnDefinitionSql(session, column, remoteColumnName));
+            execute(session, connection, sql);
+        }
+        catch (SQLException e) {
+            throw new TrinoException(JDBC_ERROR, e);
+        }
+    }
+
+    @Override
+    public void setTableComment(ConnectorSession session, JdbcTableHandle handle, Optional<String> comment)
+    {
+        String sql = format("ALTER TABLE %s COMMENT = %s", quoted(handle.asPlainTable().getRemoteTableName()), databendVarcharLiteral(comment.orElse(NO_COMMENT)));
```

Comment:
Is there no-risk of SQL injection?

Reply:
I see other module like snowflake also do like this

Reply:
Please check if we can use a prepared statement here.